compiler/src/main/java/org/robovm/compiler/config/Config.java

```diff
@@ -1361,7 +1396,12 @@ public void addPluginArgument(String argName) {
             }
             config.pluginArguments.add(argName);
         }
-
+        
+        // TODO: Are we disobeying any rules by allowing config to be fetched?  All I want is the targetBinary.
```

Comment:
I think i'd be better to have a getter in the builder. We do something similar for plugins imo.